src/mono/mono/mini/interp/transform.c

```diff
@@ -349,6 +349,63 @@ mono_mint_type (MonoType *type)
 	return -1;
 }
 
+// This doesn't allocate a new var, but marks the existing var as renamable,
+// allocating space for additional var data.
+int
+interp_create_renamable_var (TransformData *td, int var)
```

Comment:
i think this and the other one should probably not be `interp_create_...` if they are attaching information to an existing var. like `interp_make_var_renamable`